physical/raft/raft.go

```diff
@@ -130,6 +130,14 @@ type LeaderJoinInfo struct {
 	// via 'leader_api_addr'.
 	AutoJoin string `json:"auto_join"`
 
+	// AutoJoinScheme defines the optional URI protocol scheme for addresses
+	// discovered via auto-join.
+	AutoJoinScheme string `json:"auto_join_scheme"`
```

Comment:
Do we want omitempty on this and the other AutoJoin fields?

Reply:
Depends. Is this configuration exposed in any requests or other client-facing calls? I'm suspicion is that it's not. @briankassouf?

Reply:
Yeah, I don't actually think that we ever json marshal this struct, but you may want to confirm that.

Reply:
I was thinking about the sanitized config output, though perhaps it doesn't marshall this directly.